k8s/prometheus/apptest/deployer/manifest/tester.yaml.template

```diff
@@ -27,4 +27,6 @@ spec:
           fieldPath: metadata.namespace
     - name: APP_INSTANCE_NAME
       value: ${APP_INSTANCE_NAME}
+    - name: GRAFANA_GENERATED_PASSWORD
+      value: ${GRAFANA_GENERATED_PASSWORD}
```

Comment:
Use `valueFrom` instead. Example: https://github.com/GoogleCloudPlatform/click-to-deploy/pull/360/files#diff-9b5d31e09c7b5411d2b3b06269ae00b3R30